### PR TITLE
PEP-257 bodhi.client.bindings.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

File: bodhi/client/bindings.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright 2007-2016  Red Hat, Inc.
-# This file is part of bodhi
+# Copyright © 2007-2017 Red Hat, Inc. and others.
```
Comment (pypingou) [inline bodhi/client/bindings.py]: Could be 2018 now I guess :)

**Review by pypingou — APPROVED** [review]
Looks fine to me

File: bodhi/client/bindings.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/client/bindings.py]: Fixed!

**Pull request merged**